pdpserver/server/ctrl_storage.go

```diff
@@ -0,0 +1,49 @@
+package server
+
+import (
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+type storageHandler struct {
+	s *Server
+}
+
+func (handler *storageHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// sanity check
+	root := handler.s.p
+	if root == nil {
+		io.WriteString(w, "Server missing policy storage")
+		return
```

Comment:
You need to return some http error code here (like 404) with `w.WriteHeader` function.